src/overlays/actors/ovl_En_Tana/z_en_tana.c

```diff
@@ -20,13 +27,71 @@ const ActorInit En_Tana_InitVars = {
     (ActorFunc)EnTana_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Tana/EnTana_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Tana/EnTana_Destroy.s")
+static char* shelfTypes[] = {
+    "木の棚", // "Wooden Shelves"
+    "石の棚", // "Stone Shelves"
+};
+
+static const ActorFunc drawFuncs[] = {
+    (ActorFunc)func_80B17FC4,
+    (ActorFunc)func_80B1809C,
+    (ActorFunc)func_80B1809C,
+};
+
+static u32 dListTbl[] = {
```

Comment:
Since the type is known to be Gfx* it should be used
```suggestion
static Gfx* dListTbl[] = {
```